client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -194,39 +144,12 @@ const OutboundShipmentListViewComponent: FC = () => {
         modalController={modalController}
         simplifiedTabletView={simplifiedTabletView}
       />
-      {tableUsabilityImprovements ? (
-        <MaterialTable table={table} />
-      ) : (
-        <DataTable
-          id="outbound-list"
-          enableColumnSelection
-          pagination={{ ...pagination, total: data?.totalCount ?? 0 }}
-          onChangePage={updatePaginationQuery}
-          columns={columns}
-          data={data?.nodes ?? []}
-          isError={isError}
-          isLoading={isLoading}
-          noDataElement={
-            <NothingHere
-              body={t('error.no-outbound-shipments')}
-              onCreate={modalController.toggleOn}
-            />
-          }
-          onRowClick={row => {
-            navigate(row.id);
-          }}
-        />
-      )}
+      <MaterialTable table={table} />
+
       <Footer
         selectedRows={selectedRows}
-        resetRowSelection={resetRowSelection}
+        resetRowSelection={table.resetRowSelection}
       />
     </>
   );
 };
-
-export const OutboundShipmentListView: FC = () => (
-  <TableProvider createStore={createTableStore}>
```

Comment:
a nice win is that we no longer need these wrapper components for table context!